packages/app/src/components/Dimensions/DimensionLabel.js

```diff
@@ -54,29 +42,13 @@ export class DimensionLabel extends Component {
     render = () => {
         const Label = this.renderLabel();
 
-        const containerStyle =
-            this.props.isSelected && !this.props.isDeactivated
-                ? styles.selected
-                : styles.unselected;
-
-        return (
-            <div className="labelContainer" style={containerStyle}>
-                {Label}
-            </div>
-        );
+        return Label;
     };
 }
 
-const mapStateToProps = state => ({
-    currentLayout: sGetUiLayout(state),
-    items: sGetUiItems(state),
-});
-
 export default connect(
-    mapStateToProps,
+    null,
     {
         openDialog: id => acSetUiActiveModalDialog(id),
```

Comment:
Here you can use the shorthand notation:
`{openDialog: acSetUiActiveModalDialog}`